src/components/shared/LikeButton.js

```diff
@@ -0,0 +1,27 @@
+import React, {useEffect, useState} from "react";
+import FavoriteIcon from "@material-ui/icons/Favorite";
+import {addFavoriteAction, removeFromFavoritesAction} from "../../store/reducers/favoritesReducer";
+import {useDispatch, useSelector} from "react-redux";
+
+export function LikeButton({breedId}) {
+    const dispatch = useDispatch();
+
+    const addFavoriteHandler = () => {
+        dispatch(addFavoriteAction(breedId))
+    }
+    const removeFromFavoritesHandler = () => {
+        dispatch(removeFromFavoritesAction(breedId))
+    }
+    const [color, setColor] = useState('primary')
```

Comment:
You do not really need this color state, just type in 25 line color={favoriteIdsState.includes(breedId)?'secondary' : 'primary'}